### PR TITLE
Firebase ML Modify Operation Handling Code to match rpc codes not html codes

### PR DESCRIPTION
**Pull request by ifielker**
Modify Operation handling code to match rpc codes not html codes.

### CODE REVIEW

File: firebase_admin/_utils.py [hunk 1]
```diff
@@ -120,9 +139,9 @@ def handle_operation_error(error):
             message='Unknown error while making a remote service call: {0}'.format(error),
             cause=error)
 
-    status_code = error.get('code')
+    rpc_code = error.get('code')
```
Comment (hiranya911) [inline firebase_admin/_utils.py]: Doesn't this response contain a `status` field? That's how we map errors sent by other OnePlatform APIs. See `_handle_func_requests` function in this module.

https://cloud.google.com/apis/design/errors#http_mapping

Comment (ifielker) [reply firebase_admin/_utils.py]: It does for non-operations responses. But for Operation errors it does not. Just message and code (details optional and not applicable in this case)

File: firebase_admin/_utils.py [hunk 2]
```diff
@@ -59,6 +59,25 @@
 }
 
 
+_RPC_CODE_TO_ERROR_CODE = {
```
Comment (hiranya911) [inline firebase_admin/_utils.py]: Nit: May be mention https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto in a comment so we know where these mappings come from.

Comment (ifielker) [reply firebase_admin/_utils.py]: done

**Pull request merged**